LayoutTests/imported/w3c/web-platform-tests/css/css-rhythm/block-step-size-0-disables-rounding.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html>
+<head>
+<title></title>
```

Comment:
The empty titles are a bit strange in all of the WPTs

Reply:
Yes it is weird... Removed